sdk/Lusid.Sdk.Tests/tutorials/Instruments/FxOptionExamples.cs

```diff
@@ -150,5 +152,110 @@ public void FxOptionPortfolioCashFlowsExample(ModelSelection.ModelEnum model, bo
             var fxOption = InstrumentExamples.CreateExampleFxOption(isDeliveryNotCash);
             CallLusidGetPortfolioCashFlowsEndpoint(fxOption, model);
         }
+
+        /// <summary>
+        /// Lifecycle management (i.e. conservation of PV on cashflow events) for fx option
+        /// Cash-settled case:
+        ///      pv(option) = max(spot - strike, 0) before maturity and
+        ///      pv(cashflow) = max(spot_at_maturity - strike, 0) after maturity
+        /// Physically-settled case: If option is exercised, pay the strike to obtain the underlying.
+        ///      pv(option) = max(spot - strike, 0) before maturity and
+        ///      pv(option exercise payment) + pv(received underlying) = max(spot_at_maturity - strike, 0) after maturity (assuming fx rates haven't changed)
```

Comment:
not actually too sure on the wording of pv(option exercise payment) and pv(received underlying).

Reply:
just changed to pv(cash balance)